wormhole-connect/src/utils/wallet/solana.ts

```diff
@@ -70,9 +75,74 @@ export async function signAndSendTransaction(
   if (!wallet || !wallet.signAndSendTransaction) {
     throw new Error('wallet.signAndSendTransaction is undefined');
   }
+  const solanaWallet = wallet as SolanaWallet;
+  if (wallet.getName() === 'SquadsX' || wallet.getName() === 'Phantom') {
+    const tx = transaction as Transaction;
+    const { connection } = solanaContext();
+    const payer = solanaWallet.getAdapter().publicKey!;
+    const alt = new PublicKey('DUJnxSGLn7Mgh1ahP14umL52L7Z5dVpeMBsgr2bvdbHj');
+    const unique = tx.instructions
+      .flatMap((ix) => ix.keys)
+      .reduce(
+        (acc, { pubkey }) => ({
+          ...acc,
+          [pubkey.toString()]: pubkey,
+        }),
+        {},
+      );
+    console.log(Object.values<PublicKey>(unique).map((key) => key.toBase58()));
 
-  return await (wallet as SolanaWallet).signAndSendTransaction({
-    transaction: transaction as Transaction,
-    options,
-  });
+    if (connection) {
+      if (wallet.getName() === 'Phantom') {
+        const extendInstruction = AddressLookupTableProgram.extendLookupTable({
```

Comment:
this will not fly in the way it's currently implemented (writting to the hardcoded LUT above) because LUTs have a limited size (very limited for this particular solution)...
what's the plan here? what are the addresses we are putting into the LUT?